sources/api/migration/migrations/v1.7.0/add-cfsignal/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "cfsignal"
```

The migration crate name needs to be updated
```suggestion
name = "add-cfsignal"
```